File: University-Python-2023-Q4/Lab_Exercises/Lab_2/Library/person.py
class Person:
    def __init__(self, name, age, address):
        self.name = name
        self.age = age
        self.address = address

    def __str__(self):
        return f'[{self.name} - {self.age} - {self.address}]'

    def info(self):
        return f'{self.name} - {self.age} - {self.address}'

File: University-Java-2024-Q4/JavaLab/Lab7/src/Test/Person.java
package Test;

import java.util.Objects;

public class Person {
    int age;
    String name;

    Person(int age, String name){
        this.age = age;
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    @Override
    public boolean equals(Object o) {
        Person person = (Person) o;
        return age == person.age && Objects.equals(name, person.name);
    }

    @Override
    public int hashCode() {
        return Objects.hash(age, name);
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
